net/quagga/src/opnsense/mvc/app/controllers/OPNsense/Quagga/Api/DiagnosticsController.php

```diff
@@ -59,6 +59,12 @@ public function showipbgpsummaryAction()
         $response = $backend->configdRun("quagga diag-bgp summary");
         return array("response" => $response);
     }
+    public function showrunningconfigAction()
+    {
+        $backend = new Backend();
+        $response = $backend->configdRun("quagga general-runningconfig");
+        return array("response" => $response);
+    }
```

Comment:
I have made a wrapper to make this functions one line:
```php
<?php
return $this->get_general_information('runningconfig')
```

Which will do the same.